telefonica_eclipse_checkstyle.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0"?>
```

Comment:
Maybe this file was unintencionally added to the PR? (Note that we already have a telefonica_checkstyle.xml file in the repository)

Reply:
Actual checkstyle, doesn't works whit Eclipse IDE, this one was created to avoid the problems.
should we remove it?

Reply:
If telefonica_checkstyle.xml is broken, then a solution is to replace that one with yours. Does it contain the same rules?

Question: apart from Eclipse, does your telefonica_checkstyle_eclipse.xml file works with plain check style? (I mean `mvn checkstyle` or similar... I don't remember right now the exact syntax of the command...)

Reply:
Two modules throwing error on Eclipse was disabled. 
```xml
<module name="FileContentsHolder"/>
```
```xml
<module name="RedundantThrows"/>
```

Reply:
We'll take a look closely to this checkstyle plugin, for now it seems to be better to leave this file out of the PR. But thanks for the feedback.

Reply:
New Checkstyle removed 
https://github.com/Stdri/fiware-cygnus/commit/1941e81055e2d76d0b345879d0e68e244f0d241b